### PR TITLE
Remove gradle profiles

### PR DESCRIPTION
**Pull request by frankgh**
- Move gpdb properties to gradle properties
- Use jcentral as the main repository

Co-authored-by: Lav Jain <ljain@pivotal.io>
Co-authored-by: Francisco Guerrero <aguerrero@pivotal.io>

### CODE REVIEW

File: server/build.gradle [hunk 1]
```diff
@@ -356,20 +339,20 @@ task assemble(type: Copy, dependsOn: [subprojects.jar, project(':pxf-service').w
         rename('pxf-service', 'pxf')
         filter(ReplaceTokens,
                 tokens: [
-                        'pxfPortNum'    : databaseProperties.pxfPortNum,
-                        'pxfDefaultUser': (databaseProperties.pxfDefaultUser == null ? "" : databaseProperties.pxfDefaultUser)])
+                        'pxfPortNum'    : pxfPortNum,
+                        'pxfDefaultUser': pxfDefaultUser])
```
Comment (denalex) [inline server/build.gradle]: should we remove these tokens and hardcode the default values (5888, gpadmin - respectively) ?

File: server/build.gradle [hunk 2]
```diff
@@ -356,20 +339,20 @@ task assemble(type: Copy, dependsOn: [subprojects.jar, project(':pxf-service').w
...
-                tokens: ['pxfLogDir'     : databaseProperties.pxfLogDir,
-                         'pxfRunDir'     : databaseProperties.pxfRunDir,
-                         'pxfPortNum'    : databaseProperties.pxfPortNum,
-                         'pxfDefaultUser': (databaseProperties.pxfDefaultUser == null ? "" : databaseProperties.pxfDefaultUser),
-                         'pxfDefaultUserImpersonation': (databaseProperties.pxfDefaultUserImpersonation == null ? "" : databaseProperties.pxfDefaultUserImpersonation)]) into 'conf'
+                tokens: ['pxfLogDir'     : pxfLogDir,
+                         'pxfRunDir'     : pxfRunDir,
+                         'pxfPortNum'    : pxfPortNum,
+                         'pxfDefaultUser': pxfDefaultUser,
+                         'pxfImpersonation': pxfImpersonation]) into 'conf'
```
Comment (denalex) [inline server/build.gradle]: same here, remove tokens, hardcode defaults ?

File: server/gradle.properties [hunk 3]
```diff
@@ -21,11 +21,14 @@ hadoopVersion=2.8.5
...
 osFamily=el6
 parquetVersion=1.9.0
 awsJavaSdk=1.11.416
 org.gradle.daemon=true
-#org.gradle.parallel=true
+pxfLogDir=${PXF_HOME}/logs
+pxfRunDir=${PXF_HOME}/run
+pxfPortNum=5888
+pxfDefaultUser=
+pxfImpersonation=true
```
Comment (denalex) [inline server/gradle.properties]: why parallel gradle has been commented and removed ?

File: server/gradle.properties [hunk 4]
```diff
@@ -21,11 +21,14 @@ hadoopVersion=2.8.5
 htraceVersion=4.0.1-incubating
 hiveVersion=1.2.1
 hbaseVersionJar=1.1.2
-hbaseVersionRPM=1.1.2
 tomcatVersion=7.0.62
 pxfProtocolVersion=v15
 osFamily=el6
```
Comment (denalex) [inline server/gradle.properties]: this property is also probably RPM artifact

File: server/build.gradle (hunk 1 above)
Comment (frankgh) [reply server/build.gradle]: In that case, would it make more sense to hardcode it in `pxf-env.sh`? and we can get rid of `filter(ReplaceTokens(...))`

File: server/gradle.properties (hunk 3 above)
Comment (frankgh) [reply server/gradle.properties]: I believe it was hanging in one of the pipelines, @lavjain might have more details

File: server/build.gradle (hunk 1 above)
Comment (denalex) [reply server/build.gradle]: yes, that's what I meant, hardcode in the source, remove gradle props.

File: server/gradle.properties (hunk 4 above)
Comment (frankgh) [reply server/gradle.properties]: are you referring to the `osFamily` property?

Comment (denalex) [reply server/gradle.properties]: yes

Comment (frankgh) [reply server/gradle.properties]: I will search for references, and will remove if it is safe to remove

**Pull request merged**